sections/study_programs.md

```diff
@@ -0,0 +1,135 @@
+# CentrePoing Designer Study Programs
```

Comment:
(sp?) CentrePoint

Reply:
Based on review with Amulya I had removed the heading for CentrePoint Designer Study Programs. Moved the reference to Study Programs documentation to be under Export Dataset Files heading.